etl/steps/data/garden/ophi/2022-12-13/multidimensional_poverty_index.py

```diff
@@ -0,0 +1,115 @@
+"""
+This code generates the garden step with the MPI dataset for both harmonized over time and
+current margin estimates for the variables MPI, share of MPI poor and intensity of poverty.
+"""
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from owid.datautils import geo
+from structlog import get_logger
+
+from etl.helpers import Names
+from etl.paths import DATA_DIR
+
+MEADOW_VERSION = "2022-12-13"
+
+log = get_logger()
+
+# naming conventions
+N = Names(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("multidimensional_poverty_index.start")
+
+    # read dataset from meadow
+    ds_meadow = Dataset(DATA_DIR / f"meadow/ophi/{MEADOW_VERSION}/multidimensional_poverty_index")
+    tb_meadow = ds_meadow["multidimensional_poverty_index"]
+
+    df = pd.DataFrame(tb_meadow)
+
+    # %% [markdown]
+    # ### Note on `year`
+    #
+    # The way `year` is formatted – as a string variable often spanning two calendar years – won't work with our schema. We have to map the data to a single (integer) year.
+    #
+    # For now, arbitrarily, I take the first year in these cases and convert to integer.
+
+    # %%
+    # First year = first 4 characters of the year string
+    df["year"] = df["year"].str[:4].astype(int)
+
+    # %% [markdown]
+    # ## Multi-dimesional poverty measures
+    #
+    # At least initially, we will be primarily concerned with the three measures that relate to overall multi-dimensional poverty:
+    # - `Headcount ratio`: the share of population in multidimensional poverty
+    # - `Intensity`: a measure of the average depth of poverty (of the poor only – NB, not like the World Bank's poverty gap index)
+    # - `MPI`: the product of `Headcount ratio` and `Intensity`.
+    #
+    # These are multi-dimensional poverty measures – a weighted aggregation across many individual indicators.
+    # Here I prepare this data as I would for uploading to OWID grapher and visualize it – separating the `hot` from the `cme` data.
+
+    # %%
+    # Prep data for OWID
```

Comment:
By reading the comments, it sounds as if you have to prepare the data for OWID or for grapher. But a priori the goal of garden is neither of those. The goal is to clean and harmonize the data. I think saying "Prep data for OWID" sounds as if our data need some special treatment, which is not the case.

Reply:
Sure, in that case was inherited from old pre-ETL code